demo/src/Components/BottomSheet/BottomSheet.tsx

```diff
@@ -17,74 +40,154 @@ const BottomSheet = (props: HvComponentProps) => {
     animationDuration: animationDuration
       ? parseInt(animationDuration, 10)
       : 250,
+    contentSections: props.element.getElementsByTagNameNS(
+      namespace,
+      BottomSheetContentSection.localName,
+    ),
     dismissible:
       props.element.getAttributeNS(namespace, 'dismissible') !== 'false',
+    stopPoints: props.element.getElementsByTagNameNS(
+      namespace,
+      BottomSheetStopPoint.localName,
+    ),
+    swipeToClose:
+      props.element.getAttributeNS(namespace, 'swipe-to-close') === 'true',
     toggleEventName: props.element.getAttributeNS(
       namespace,
       'toggle-event-name',
     ),
     visible: props.element.getAttributeNS(namespace, 'visible') === 'true',
   };
+  const hvStyles: HvStyles = {
+    bottomSheetBackgroundColor:
+      Hyperview.createStyleProp(props.element, props.stylesheets, {
+        ...props.options,
+        styleAttr: 'bottom-sheet:container-style',
+      })[0]?.backgroundColor || '#fff',
```

Comment:
this approach is a bit restrictive, as we assume only the `backgroundColor` would be set on the `bottom-sheet:container-style` attribute.
Instead, it would make more sense to expose all styles, i.e.

```suggestion
    containerStyle: Hyperview.createStyleProp(props.element, props.stylesheets, {
      ...props.options,
      styleAttr: 'bottom-sheet:container-style',
    })
```

then, where used, combine the default with these, i.e.

```
         [
          styles.bottomSheetContainer,
          bottomSheetStyle,
          hvStyles.containerStyle,
          {
            height: SCREEN_HEIGHT,
            top: SCREEN_HEIGHT,
          },
```

note that the first dict of the list probably should come from a defined style in styles.ts.